Lib/fontTools/misc/plistlib.py

```diff
@@ -513,13 +533,24 @@ def dump(
     Args:
         value: An object to write.
         fp: A file opened for writing.
-        sort_keys: Whether keys of dictionaries should be sorted.
-        skipkeys: Keys which should be skipped when enumerating.
-        use_builtin_types: Whether to use standard plist types or
-            to wrap values in a Data block. Deprecated.
-        pretty_print: Whether to indent the output.
+        sort_keys (bool): Whether keys of dictionaries should be sorted.
+        skipkeys (bool): Whether to silently skip non-string dictionary
+            keys.
+        use_builtin_types (bool): If true, byte strings will be
+            encoded in Base-64 and wrapped in a ``data`` tag; if
+            false, they will be either stored as strings or an
```

Comment:
```suggestion
            false, they will be either stored as ASCII strings or an
```

same suggestions as above about "represented" -> "decided as ASCII"